packages/dashmate/docker-compose.yml

```diff
@@ -40,25 +40,6 @@ services:
       - core
       - masternode
```

Comment:
You don't need profile masternode anymore. Please check the generateEnvs function and update logic there as well